forms-flow-api/src/formsflow_api/resources/process.py

```diff
@@ -232,3 +245,33 @@ def delete(process_id: int):
         """Delete process data by id."""
         response, status = ProcessService.delete_process(process_id), HTTPStatus.OK
         return response, status
+
+@cors_preflight("GET, OPTIONS")
+@API.route("/process-history/<string:process_name>", methods=["GET", "OPTIONS"])
```

Comment:
For better resource reading, can we have the endpoint /process/<process_name>/versions ?

Reply:
updated